test/e2e/config/nutanix.yaml

```diff
@@ -13,6 +13,12 @@ images:
     loadBehavior: mustLoad
   # ## PLEASE KEEP THESE UP TO DATE WITH THE COMPONENTS
   # Cluster API v1beta1 Preloads
+  - name: registry.k8s.io/cluster-api/cluster-api-controller:v1.3.5
```

Comment:
Let's change controller versions we pre-fetch to v1.4.9, v1.5.4, and v1.6.0

Reply:
https://github.com/kubernetes-sigs/cluster-api/releases

Reply:
Note: We can take this in a separate PR.

Reply:
Done